data/mods/ssb/abilities.ts

```diff
@@ -37,7 +37,15 @@ export function changeSet(context: Battle, pokemon: Pokemon, newSet: SSBSet) {
 	let item = newSet.item;
 	if (typeof item !== 'string') item = item[Math.floor(Math.random() * item.length)];
 	pokemon.setItem(item);
+	changeMoves(context, pokemon, newSet);
+}
 
+/**
+ * Assigns new moves to a Pokemon
+ * @param pokemon The Pokemon whose moveset is to be modified
+ * @param newSet The set whose moves should be assigned
+ */
+export function changeMoves(context: Battle, pokemon: Pokemon, newSet: SSBSet) {
```

Comment:
Im gonna be honest, this looks incredibly niche as youre the only one who is using this

Reply:
I didn't want to duplicate code between `changeSet` and Refactor, though.

Reply:
(`changeSet` calls `changeMoves`)

Reply:
Gotcha